fuzzers/libfuzzer_libmozjpeg/Makefile

```diff
@@ -18,8 +18,8 @@ libafl_cxx: target/release/libafl_cxx
 libafl_cc: target/release/libafl_cxx
 
 mozjpeg-4.0.3-compiled: mozjpeg-4.0.3 libafl_cc
-	cd $(PWD)/mozjpeg-4.0.3 && cmake --disable-shared . -DCMAKE_C_COMPILER=$(abspath $(PWD)/../target/release/libafl_cc) -DCMAKE_CXX_COMPILER=$(abspath $(PWD)/../target/release/libafl_cxx) -G "Unix Makefiles"
```

Comment:
@domenukk this is not how to write makefiles, it is broken

Reply:
$(PWD) is correct

Reply:
or better, PWD = $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))

Reply:
`PWD` didn't work on Ubuntu in some occasions, according to ci. maybe that was in the sh file though, not sure

Reply:
Realpath doesn't exist on MacOS, don't use it

Reply:
realpath is a makefile command, if make exists realpath exists. Btw macos has realpath it is part of coreutils

Reply:
It doesn't matter what it's part of, MacOS doesn't have it.
https://stackoverflow.com/questions/3572030/bash-script-absolute-path-with-os-x

Reply:
If it's a makefile command then we can use it in makefiles, but use $(pwd) in shell scripts, I can check.